# Estrutura Condicional

`if`, `elif`, `else` (**se**, **senão se**, **senão**)

In [ ]:
#Sinal de um número:

a =int( input("Digite um número: "))

if a>0:
    print("Positivo.")
elif a==0:
    print("Nulo.")
else:
    print("Negativo")

Digite um número: -8
Negativo


### Vamos construir uma estrutura condicional?

### Baskara 

Um polinômio de segundo grau é dado por

$$\displaystyle P(x) = ax^2 + bx + c $$

A raiz de um polinômio é dado quando


$$\displaystyle ax^2 + bx + c = 0$$

ao usar o método de completar quadrados podemos chegar na seguinte solução 

$$\displaystyle\mathbf{x} = \frac{- b \pm \sqrt{\Delta}}{2 \cdot a} $$ com $ \Delta = b^2 - 4 \cdot a \cdot c $

quando o $\displaystyle\Delta > 0$, o polinômio possui duas raizes reais e quando o $\displaystyle\Delta < 0 $ , o polinômio possui duas raizes complexos. Por fim, se $\displaystyle\Delta = 0 $, o polinômio possui apenas uma raiz real.

In [ ]:
#Definindo os coeficientes da equação e verificando o delta, obtemos
a,b,c = 1,-1,-1

delta = b**2-4*a*c

if delta > 0:
    print("O polinômio possui duas raizes reais.")
elif delta < 0:
    print("O polinômio possui duas raizes complexos")
else:
    print("O polinômio possui apenas uma raiz real.")

#Primeira raiz:
x1 = (- b + (delta)**0.5)/(2*a)

#Segunda raiz:
x2 = (- b - (delta)**0.5)/(2*a)

if x1==x2:
    print("A raiz do polinômio é ", x1)
else:
    print("As raizes do polinômio são {} e {}".format(x1,x2))

O polinômio possui duas raizes reais.
As raizes do polinômio são 1.618033988749895 e -0.6180339887498949


In [ ]:
phi = x1
print(x1)

1.618033988749895


$$\mathbf{\phi} = 1.618033988749895 $$

Considerado um símbolo de perfeição na natureza e nas construções, chegando a receber o nome de proporção divina.

![5aef45c04a46032238f490b0997d183e.jpg](attachment:5aef45c04a46032238f490b0997d183e.jpg)

In [ ]:
'''Atividade:Montar uma calculadora que recebe dois números, 
se o primeiro número é maior que o segundo número, a calculadora irá calcular uma subtração entre o primeiro e o segundo. 
Caso contrário,  a calculadora irá somar os números. Se os números forem iguais, irão multiplicar.'''

num1 = int(input("Digite o primeiro número: "))
num2 = int(input("Digite o segundo número: "))

if num1>num2:
    print(num1-num2)
elif num1<num2:
    print(num1+num2)
else:
    print(num1*num2)

Digite o primeiro número: 5
Digite o segundo número: 5
25


In [ ]:
#P/fazer Criar uma organizadora que recebe três números e organiza na ordem crescente.

#P/Fazer   Desafio: Criar uma calculadora que encontra as raizes de um polinômio de terceiro grau.